miasm2/jitter/jitcore.py

```diff
@@ -160,7 +161,7 @@ def jit_call(self, label, cpu, vmmngr):
         """
 
         fc_ptr = self.lbl2jitbloc[label]
-        return self.exec_wrapper(fc_ptr, cpu.cpu, vmmngr.vmmngr)
+        return self.exec_wrapper(fc_ptr, cpu)
```

Comment:
Can you update `jitter/Jitllvm.c:llvm_exec_bloc` accordingly ?

Even if it's no supported right now, it will avoid later errors (and it's related to this patch).